pkg/kn/commands/types_test.go

```diff
@@ -70,6 +109,11 @@ func TestGetClientConfig(t *testing.T) {
 
 	multiConfigs = multiConfigs
 	for _, tc := range []typeTestCase{
+		{
+			"",
+			clientcmd.NewDefaultClientConfigLoadingRules().ExplicitPath,
+			"",
+		},
```

Comment:
What's the difference to the test case below ?

Reply:
Oops. Accidentally left a thing in from messing around.